nms/app/views/events/EventsTable.tsx

```diff
@@ -70,33 +58,39 @@ const useStyles = makeStyles(theme => ({
   },
 }));
 
-function getEventDescription(event) {
+function getEventDescription(event: {event_type: string}) {
   switch (event.event_type) {
     case 'processed_updates':
       return 'Updates streamed from orchestrator were processed by the gateway';
+
```

Comment:
Usually I'm a fan of blank line to group things, but here it doesn't feel right.